documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,147 @@
+. Set the `Packaging` option to pom as follows:
+.. In the `Project Explorer` view, expand your existing Maven project and double-click `pom.xml` in the expanded view.
+.. The `Overview` tab appears with configuration information for your Maven project.
+.. In the `Packaging` drop-down menu, select the `pom` option.
++
+.Set the Project's Packaging Option
+image::mvnbasic_set_packaging.png[Set the Project's Packaging Option]
++
+.. Click the save icon to save the changes.
+
+=== Creating a New Maven Module
+. Create a new Maven module as follows:
+.. In your workspace, navigate to menu:File[New > Other].
+.. From the wizard selection screen, expand `Maven` and select `Maven Module`.
+.. Click `Next >` to continue.
+. In the first step of the `New Maven Module` wizard, enter the initial module details:
+.. Enter the desired module name in the `Module Name` field.
+.. Click `Browse` from the `Parent Project` field to select an existing Maven project from the displayed list, then click `OK`.
+.. The following optional steps are available for further configuration, if required:
+... (Optional) Select the `Create a simple project (skip archetype selection)` check box. If this check box is selected, the `Select an Archetype` step of the wizard is skipped.
+... (Optional) Select the `Add project(s) to working set` check box to add additional projects to the working set.
+... (Optional) Click `Advanced` to view additional optional advanced configuration for the new Maven project, such as whether to `Resolve Workspace projects`, `Profiles`, and any `Name templates` to set a desired naming convention for the project.
++
+.Set the Module Name and Parent
+image::mvnbasic_module_configuration.png[Set the Module Name and Parent]
++
+.. When the configuration is complete, click `Next` to continue.
+. In the second step of the `New Maven Module` wizard, enter the module archetype information:
+.. Ensure that the `Show the last version of Archetype only` check box is selected. This ensures that only the latest version of each archetype displays.
+.. Select an archetype based on the purpose of the project you are creating. Use the keyword `maven-archetype-quickstart` in the `Filter` field to locate a sample Maven project archetype.
++
+.Select a Module Archetype
+image::mvnbasic_select_module_archetype.png[Select a Module Archetype]
++
+.. Click `Next` to continue.
+. In the third step of the `New Maven Module` wizard, enter the archetype details:
+.. Add a unique group ID to the `Group Id` field. Note that this ID must be different from the group ID of the included Maven project.
+.. Add the desired version number in the `Version` field. As an example, the version `0.0.1-SNAPSHOT` was used as the version number for the previously created Maven project, therefore here we can use a variation such as `0.1.1-SNAPSHOT`.
```

Comment:
Changed this to:

"Add the desired version number in the Version field. For details about the appropriate version build numbers, see Project Versions[link]"